### PR TITLE
[MPQEditor] IntroduceMPQSelectionPanel

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit introduces MPQSelectionPanel for manual mixed precision quantization task.

Its correctness is tested in https://github.com/Samsung/ONE-vscode/pull/1530

Fresh draft: https://github.com/Samsung/ONE-vscode/pull/1530
Full draft: https://github.com/Samsung/ONE-vscode/pull/1505
Related: https://github.com/Samsung/ONE-vscode/issues/1491

ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

File: src/MPQEditor/MPQCircleSelector.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  document: vscode.TextDocument;
+  names: any;
+  panel: vscode.WebviewPanel;
+};
+
+export class MPQSelectionPanel
+  extends CircleGraphCtrl
+  implements CircleGraphEvent
+{
+  public static readonly viewType = "one.viewer.mpq";
```
Comment (dayo09) [inline src/MPQEditor/MPQCircleSelector.ts:41]: I wonder if it's 'viewer' or' 'editor'..?

Comment (stamalakhov) [reply src/MPQEditor/MPQCircleSelector.ts:41]: @dayo09 
For now its viewer with 'selector' abilities. I mean it doesn't have a document to edit. IMHO.

**Review by dayo09 — APPROVED** [review]

File: src/MPQEditor/MPQCircleSelector.ts (hunk 1 above)
Comment (dayo09) [reply src/MPQEditor/MPQCircleSelector.ts:41]: Oh, Thanks for the explanation!

**Pull request merged**